src/Symfony/Component/Form/Form.php

```diff
@@ -766,10 +766,28 @@ public function getClickedButton()
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getAllErrors()
+    {
+        return $this->getErrorIterator(true, true)->all();
+    }
+
     /**
      * {@inheritdoc}
      */
     public function getErrors($deep = false, $flatten = true)
+    {
+        @trigger_error('Calling «getErrors» is deprecated.  Please use either getErrorIterator or getAllErrors', E_USER_DEPRECATED);
```

Comment:
The deprecation message should be improved IMO

Reply:
I see 3 changes to do:

- the method name should be wrapped in double quotes, not in `«` to play well with our rendering in the profiler
- the message should mention the version deprecating it, as done elsewhere
- we should probably inspect the arguments being passed to suggest the right replacement, instead of suggesting both.